### DISCUSSION TITLE
Patching Zephyr code in custom project

### DISCUSSION THREAD

**Discussion by pres2300** [zephyrproject-rtos/zephyr#62870] [Q&A] [answered]
While I don't think the exact details are necessary for discussion, I'm working on a project with a Nordic nRF52840.  There are some changes for our specific application that make sense to make in some lover level driver code of Zephyr.  Is there any recommended way to manage this?  We use `west` to retrieve the nRF SDK that includes Zephyr.  It would be nice to then apply a patch to the code at this point.  That way we don't need to host the entire SDK in our own repo and can just track the patch.

One way I could do this is creating a custom command for west that is run as part of our project onboarding.  Is there a better way to do this?

**Comment by yashi** [comment]
How about using your own Zephyr repository with your fix committed.

**Comment by nordicjm** [comment, marked as answer]
>  It would be nice to then apply a patch to the code at this point. That way we don't need to host the entire SDK in our own repo and can just track the patch.

Not supported, fork the repo (you can minimise repo size by only forking the branch you need instead of all branches) then use the forked repo in your manifest
